recipes/r-spgwr/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set version = '0.6-37' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-spgwr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/spgwr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/spgwr/spgwr_{{ version }}.tar.gz
+  sha256: d357d84b957ab5808230bb13fa65beebef730fd4d703ab5604307218fd22b4d3
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
```

Needs `stdlib` declarations:

```suggestion
    - {{ compiler('c') }}        # [not win]
    - {{ stdlib('c') }}          # [not win]
    - {{ compiler('m2w64_c') }}  # [not win]
    - {{ stdlib('m2w64_c') }}    # [not win]
```